bot.py

```diff
@@ -159,9 +157,9 @@ def run_if_author_is_admin(interaction: discord.Interaction, function: Callable[
         return "You don't have permission to do that"
 
 
-def change_config_value(interaction: discord.Interaction, config_key: str, new_value: float) -> None:
+def change_config_value(interaction: discord.Interaction, config_key: str, new_value: Any) -> None:
```

Comment:
It has to be Any?

Reply:
yeah because it's now used to update the chance thresholds (floats) and the bypass prefix (string)

Reply:
I should probably add some docstrings soon lol